src/material/datepicker/date-range-input-parts.ts

```diff
@@ -86,17 +90,20 @@ abstract class MatDateRangeInputPartBase<D>
   protected abstract override _assignValueToModel(value: D | null): void;
   protected abstract override _getValueFromModel(modelValue: DateRange<D>): D | null;
 
+  protected readonly _dir = inject(Directionality, InjectFlags.Optional);
+  private readonly _onKeyupSubject = new Subject<KeyboardEvent>();
+
   constructor(
     @Inject(MAT_DATE_RANGE_INPUT_PARENT) public _rangeInput: MatDateRangeInputParent<D>,
-    elementRef: ElementRef<HTMLInputElement>,
+    public override _elementRef: ElementRef<HTMLInputElement>,
```

Comment:
was this private on the super class or something?

Reply:
It was protected. I need it public so that start can access the field on end and vice versa.